src/doc/style-guide/src/expressions.md

```diff
@@ -815,13 +841,12 @@ let arr = [combinable(
 )];
 ```
 
-Such behaviour should extend recursively, however, tools may choose to limit the
-depth of nesting.
+Apply this behavior recursively.
```

Comment:
Recognize this is part of removing a "may" and references to tools, though I'm slightly paranoid that there may be cases where rustfmt used the existing language to break the recursion which may no longer be correct with the proposed changes.

I'm not quite sure how to handle this one. Thoughts?